packages/mobile/src/utils/useClipboard.ts

```diff
@@ -1,17 +1,25 @@
+import Clipboard from '@react-native-community/clipboard'
 import { useEffect, useState } from 'react'
-import { AppState, Clipboard } from 'react-native'
+import { AppState, Platform } from 'react-native'
 import Logger from 'src/utils/Logger'
 
 const CLIPBOARD_CHECK_INTERVAL = 1000 // 1sec
 
-export function useClipboard(): string {
+export function useClipboard(): [boolean, string] {
+  const [forceShowingPasteIcon, setForceShowingPasteIcon] = useState(false)
   const [clipboardContent, setClipboardContent] = useState('')
 
   useEffect(() => {
     let isMounted = true
 
     async function checkClipboardContent() {
       try {
+        const majorVersionIOS = parseInt(Platform.Version.toString(), 10)
```

Comment:
noticing that this line and the following  lines are duplicated in withPasteAware.

Reply:
Added a helper method to reuse logic :)